test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -2446,6 +2459,37 @@ static void crypto_test_check_alg_sha512(void)
 		  false);
 }
 
+static odp_queue_t sched_compl_queue_create(void)
+{
+	odp_queue_param_t qparam;
+	odp_queue_t out_queue;
+
+	odp_queue_param_init(&qparam);
+	qparam.type = ODP_QUEUE_TYPE_SCHED;
+	qparam.sched.prio  = ODP_SCHED_PRIO_DEFAULT;
+	qparam.sched.sync  = ODP_SCHED_SYNC_PARALLEL;
+	qparam.sched.group = ODP_SCHED_GROUP_ALL;
+
+	out_queue = odp_queue_create("crypto-out", &qparam);
```

Comment:
out_queue is not needed, just return odp_queue_create()

Reply:
Ack.